.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,31 +1,34 @@
 ---
-name: Bug report
+name: Simple Report Bug Report 🐛
```

Comment:
obviously non-blocking but this has a space between _Simple_ and _Report_ and none of the others do